README.md

```diff
@@ -21,21 +21,15 @@ end-to-end demo has only been tested on Linux.
 If you don't have a Linux box handy, you can [view the generated
 output](http://google.github.io/rappor/examples/report.html).
 
-To get your feet wet, install the R dependencies (details below).  It should
-look something like this:
-
-    $ R
-    ...
-    > install.packages(c('glmnet', 'optparse', 'ggplot2'))
-
-Then run:
-
-    $ ./demo.sh build  # optional speedup, it's OK for now if it fails
-
+To setup your enviroment there are some packages and R dependencies. There is a setup script to install them:
+    $ ./setup.sh
+Then to build the native components run:
+    $ ./build.sh 
 This compiles and tests the `fastrand` C extension module for Python, which
 speeds up the simulation.
 
-    $ ./demo.sh run
+Finally to run the demo run:
+    $ ./demo.sh
```

Comment:
Two issues. 
1. Currently ./demo.sh fails with "./regtest.sh: line 334: $5: unbound variable" 
2. There's inconsistency between this line and the comments inside demo.sh that stipulate that it be run "./demo.sh run"